### PR TITLE
Applying some design amongst other things

### PR DESCRIPTION
**Pull request by chris-pearce**
@davislurve take a look when you can 👍. 

I completely changed the design and added Shell, and a proper CSS structure.

Design should be treated as a WIP as I expect it'll just change organically, so things like the gradient, fonts, etc, can be changed. This was more about getting something up under the domain via GH pages (BTW if you want to add a script for this in `npm scripts` that would be sweet).

I had some fun with `clip-path`, I think we should be quite experimental and playful with the design.

Here's a Quicktime of it: https://www.dropbox.com/s/ye2z1msjpuklnhz/FED%20%7C%7C%20DEAD%20design%20movie%20clip.mov?dl=0.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,20 +1,30 @@
...
-    "build": "npm run build:css && npm run compile"
+    "svg-min": "svgo -f src/_assets/img -o dist/img",
+    "compile-html": "mustache src/data/posts.json src/index.mustache > dist/index.html",
+    "compile-scss": "node-sass --output-style compressed -o dist/css src/_assets/scss/style.scss",
+    "autoprefixer": "postcss -u autoprefixer -r dist/css/*",
+    "watch:css": "onchange 'src/_assets/scss/**.scss' -- npm run build:css",
+    "watch:html": "onchange 'src/**.mustache' -- npm run compile-html",
+    "watch:all": "parallelshell 'npm run watch:css' 'npm run watch:html'",
+    "build:css": "parallelshell 'npm run compile-scss' 'npm run autoprefixer'",
+    "build:all": "parallelshell 'npm run build:css' 'npm run compile-html' 'npm run svg-min'"
```
Comment (DaveOrDead) [inline package.json]: What's the advantage of parallelshell over &&?

File: package.json [hunk 2]
```diff
@@ -1,20 +1,30 @@
...
   },
   "devDependencies": {
     "autoprefixer": "^6.3.3",
     "mustache": "^2.2.1",
     "node-sass": "^3.4.2",
     "onchange": "^2.0.0",
+    "parallelshell": "^2.0.0",
     "postcss-cli": "^2.5.1",
-    "shell-csslib": "^0.5.0"
+    "shell-csslib": "^0.5.0",
```
Comment (DaveOrDead) [inline package.json]: Should we remove this? Or do you think we'll need it later

File: src/_assets/scss/style.scss [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+ul,
+ol {
+    list-style-position: outside;
+    margin-left: 0;
+    padding-left: 0;
+}
+
+h1 {
+    font-size: 2.1em;
```
Comment (DaveOrDead) [inline src/_assets/scss/style.scss]: Why em over rem?

File: src/_assets/scss/style.scss [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+*::after {
+    box-sizing: inherit;
+}
+
+html,
+body {
+    border: 0;
+    margin: 0;
+    padding: 0;
+    min-height: 100vh;
```
Comment (DaveOrDead) [inline src/_assets/scss/style.scss]: Are we sticking to alphabetical properties for this?

File: .gitignore [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # Specific files #
 ###################
-
+dist/
```
Comment (DaveOrDead) [inline .gitignore:4]: Maybe this should be under the Compied Source or an Output title?

File: src/_assets/scss/style.scss [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+ */
+
+.l-header {
+    background-color: $color-text-base;
+    color: #fff;
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+    justify-content: center;
+    padding: 2em calc(2em + 71px) 2em 2.5em;
```
Comment (DaveOrDead) [inline src/_assets/scss/style.scss]: Should the 71px be a setting

File: src/_assets/scss/style.scss [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+ * Logo.
+ */
+
+.l-header__logo {
+    position: absolute;
+    right: 0;
+    top: 50%;
+    transform: translateY(-50%);
+    width: 143px;
+    margin-right: -71px;
```
Comment (DaveOrDead) [inline src/_assets/scss/style.scss]: Should be alphabetical

File: src/_assets/scss/style.scss [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+.l-container {
+    margin-left: $l-header-width;
+    padding-top: 3em;
+    padding-bottom: 3em;
+    width: 100%;
+
+    > section {
+        margin-left: auto;
+        margin-right: auto;
+        max-width: 70ch;
```
Comment (DaveOrDead) [inline src/_assets/scss/style.scss]: Not seen ch units before, just had a quick read, seem cool

File: src/index.mustache [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+        {{!-- Posts --}}
+        <section class="l-posts">
+            <h1>Writing</h1>
+            <ol class="c-posts">
+                {{#posts}}
+                <li class="c-posts__item">
+                    <a href="{{url}}" rel="external" class="c-posts__link">
+                        <h2 class="c-posts__title">{{title}}</h2>
+                        <p class="c-posts__meta">
+                            <span class="c-posts__meta__label">Author:</span>
```
Comment (DaveOrDead) [inline src/index.mustache]: Not sure about these double sets of __. Will have a think on this

**Comment by DaveOrDead** [comment]
Left a bunch of comments, wasn't sure really how much detail you / we are looking for on a small standalone site so a lot of it may be overkill depending on how strict We wanted to be for this project.

Design wise the logo looks a little weird to me where it's placed and the colour splitting doesn't quite work in the way I imagined it in my head. Be interested to hear what the other guys think

File: .gitignore (hunk 5 above)
Comment (chris-pearce) [inline .gitignore:4]: I'll move it there.

File: package.json (hunk 1 above)
Comment (chris-pearce) [inline package.json]: `&&` doesn't work properly with the `watch` scripts, I was stuck in loops, see: https://css-tricks.com/why-npm-scripts/#article-header-id-16.

File: package.json (hunk 2 above)
Comment (chris-pearce) [inline package.json]: I'd say leave it for now.

File: src/_assets/scss/style.scss (hunk 4 above)
Comment (chris-pearce) [inline src/_assets/scss/style.scss]: Yeah, I'll bring in CM's stylelint (once it's done) as an NPM module, if that's cool?

File: src/_assets/scss/style.scss (hunk 3 above)
Comment (chris-pearce) [inline src/_assets/scss/style.scss]: So I guess I was just whipping something up quickly and just stuck with `em`s as it was already there. But we should start to apply an architecture and follow guidelines. I might set that up as part of this PR.

File: src/_assets/scss/style.scss (hunk 6 above)
Comment (chris-pearce) [inline src/_assets/scss/style.scss]: I wanted to but then it won't work with `calc` due to the setting be a **pre-processor** thing.

File: src/_assets/scss/style.scss (hunk 8 above)
Comment (chris-pearce) [inline src/_assets/scss/style.scss]: They're cool.

File: src/_assets/scss/style.scss (hunk 7 above)
Comment (chris-pearce) [inline src/_assets/scss/style.scss]: Will fix, see comment about stylelint.

**Pull request merged**